docs/HowTos/HOWTO-miscellaneous.md

```diff
@@ -15,10 +15,36 @@ The container-based Matomo installation requires a walk through of the web UI to
 
 ## Troubleshooting
 
-It **is** possible to SSH into the Matomo container while it is running in ECS:
+It **is** possible to SSH into the Matomo container while it is running in ECS:  `Ensure you have the latest Session Manager plugin installed if you have errors or no output to the following command`
```

Comment:
Minor quibble here -- use either * (italic) or ** (bold) instead of ` (code) for the emphasis of this sentence.

Reply:
Fixed...new approval is needed.  :grin: